### DISCUSSION TITLE
Firefly Desktop 2.0.0 Beta 4

### DISCUSSION THREAD

**Discussion by nicole-obrien** [iotaledger/firefly#4229] [Announcements]
### Changelog
------

**Change log**
- Fix recipient input close issue
- Add filter to show previously rejected transactions
- Add filter to filter activities by transactions
- Fix token info showing undefined after hiding
- Refactor network configuration settings
	- Refactor logic
	- Simplify network config options ux
	- Enable ability to set a primary node to be used by default
	- Enable ability to exclude node from list without removing it
- Refactor all onboarding logic and ux
- Improve activity generation from minting transactions
- Update [wallet.rs](http://wallet.rs/) dependency for breaking change 

  **Known issues:**
- Sending 0 value transactions with metadata and tag fails
- Reseting asset metadata may crash the profile until restart
- Storage deposit calculation for minting transaction is wrong

[Full changelog
](https://github.com/iotaledger/firefly/compare/desktop-2.0.0-beta-3...desktop-2.0.0-beta-4)

### File Hashes
[How to verify the authenticity of your Firefly Desktop download](https://wiki.iota.org/chrysalis-docs/firefly/verify_download)
| File | Platform | SHA256 Hash |
| --- | --- | --- |
| firefly-desktop-2.0.0-beta-4.exe | Windows | 96a360823d8ad237c8f182bbc7334e037b54c67d1c77935c73340a3515e4fd60 |
| firefly-desktop-2.0.0-beta-4.AppImage | Linux | 9bdd9d6a094ac3995ff9d51e1edd7a422da4e3c74422905f0bd1f575c6049789 |
| firefly-desktop-2.0.0-beta-4.dmg | MacOS |  9ae12c25526af10a56a30b7252ae9fa5eaedaded54e045615b1c8a523a01891e |

<hr /><em>This discussion was created from the release <a href='https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.0-beta-4'>Firefly Desktop 2.0.0 Beta 4</a>.</em>

**Comment by a673130565** [comment]
Windows 7 is not available, please debug well. Every version is the same problem。

**Reply by no8ody** [reply]
Windows 7 is no longer supported as far as I know and should generally not be used anymore.

**Reply by a673130565** [reply]
Okay, I'm still using Windows 7。😢
